### PR TITLE
CNV-2692

### PR DESCRIPTION
**Pull request by bgaydosrh**
This PR contains a new assembly and one new module that instructs the user how to access the Overview Dashboard displaying various information about CNV clusters.

Previously this dashboard was accessed by a URL. It can now be accessed directly through the Open Source Web Console.

See story https://jira.coreos.com/browse/CNV-2692 for details.

Need Peer Review from any of the following: @ahardin-rh @jboxman @sheriff-rh 

Please label Needs Peer Review, OpenShift-Enterprise-4.2

Thanks,
Bob

### CODE REVIEW

**Comment by bgaydosrh** [comment]
Here is a test build: http://file.bos.redhat.com/bgaydos/091919/cnv/cnv_users_guide/cnv-dashboard.html

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ousleyp) [inline cnv/cnv_users_guide/cnv-dashboard.adoc]: We're not supposed to abbreviate it to CNV. I would just call it the (cluster?) Overview Dashboard, if that's its official name. I'm also not sure if Overview Dashboard or Cluster Overview Dashboard should be capitalized or not.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ousleyp) [inline cnv/cnv_users_guide/cnv-dashboard.adoc]: 1. "Web Console" shouldn't be capitalized
2. Did you mean to refer to the dashboard here (as opposed to the web console)?

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ousleyp) [inline cnv/cnv_users_guide/cnv-dashboard.adoc]: 1. The variable for OCP is `{product-title}`, not `{product-name}`
2. Again, I don't think that we capitalize "web console"

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ousleyp) [inline cnv/cnv_users_guide/cnv-dashboard.adoc]: I think that the period should go inside the quotation marks here, since it's not code.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ousleyp) [inline modules/cnv-web-console-overview-dashboard.adoc]: Please update the assembly that this is included in

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ousleyp) [inline modules/cnv-web-console-overview-dashboard.adoc]: 1. Should it say "cluster overview dashboard" or just "overview dashboard"?
2. Same question about the capitalization, just in case. <rimshot> 😂

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ousleyp) [inline modules/cnv-web-console-overview-dashboard.adoc]: I think you mean to say OpenShift Container Platform here, right? I would just change this line to:
`** {product-title} version`

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ousleyp) [inline modules/cnv-web-console-overview-dashboard.adoc]: the word "pod" should be capitalized = "Pod"

(see https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)

**Review by ousleyp — CHANGES_REQUESTED** [review]
This isn't meant to be the official peer review; I just did a quick once-over. It largely LGTM with some issues that I identified. Thanks, Bob!

**Comment by bgaydosrh** [comment]
> This isn't meant to be the official peer review; I just did a quick once-over. It largely LGTM with some issues that I identified. Thanks, Bob!

Thanks Pan! I have to get Ashley to do the Peer Review I think as I need to verify the Web Console topic should not change anyway. Good catches, I appreciate that!

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ahardin-rh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: correct; web console is in lowercase

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (ahardin-rh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: we should actually underscore cards since we're introducing a new term:

    or _cards_.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ahardin-rh) [inline modules/cnv-web-console-overview-dashboard.adoc]: Perhaps take the terms out of quotations and embolden them instead?

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (ahardin-rh) [inline modules/cnv-web-console-overview-dashboard.adoc]: The *Utilization* card 
The *Events* card
The *Top Consumers* card

**Comment by ahardin-rh** [comment]
@bgaydosrh Just a few small comments from me. Otherwise, looks good! :rocket:

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (vikram-redhat) [inline modules/cnv-web-console-overview-dashboard.adoc]: "an brief" Typo

**Comment by rawagner** [comment]
It seems that this doc should go directly to openshift docs, not CNV docs. Overview Dashboard is part of core Openshift UI. When CNV is installed, it will extend Overview Dashboard for CNV specific parts like CNV subsystem health and VMs in Inventory Card

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (rawagner) [reply modules/cnv-web-console-overview-dashboard.adoc]: See All button is present only when some additional subsystem is present - core Openshift UI does not have See All button.
When some additional subsystem is present (ie CNV is installed) then the main health status is aggregation of all subsystems (core Openshift + CNV + ...). If user wants to examine status of particular subsystem, he can click See All

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (aburdenthehand) [inline cnv/cnv_users_guide/cnv-dashboard.adoc]: This is a bit of a nit but it's good practice for the id to correlate to the title. It makes it easier to navigate content in our our repo (especially when cross-referencing from published doc), and is becoming increasingly important as our content grows. If you do decide to update the id, don't forget to update the filename to match.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (aburdenthehand) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: "dashboard" was previously capitalised.
I would drop "sections or" since they are only referred to as "cards", and that term is introduced in markup.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (aburdenthehand) [reply modules/cnv-web-console-overview-dashboard.adoc]: Using the [OCP docs standard](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) the title should be something like 'About the Overview Dashboard' (Pan's question about capitals still stands :)) 
The module id should match the title, and also *needs* to match the filename. The `cnv-` prefix is important.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (aburdenthehand) [reply modules/cnv-web-console-overview-dashboard.adoc]: Not having 'The' before the card name makes it weird to read. Applies to all.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (aburdenthehand) [inline modules/cnv-web-console-overview-dashboard.adoc]: Is the right name for the card? I've only seen it as *Cluster Inventory*. This also applies to *Health*, *Utilization*, and *Capacity*.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (aburdenthehand) [reply modules/cnv-web-console-overview-dashboard.adoc]: Since these are GUI titles, I would use what is displayed and markup appropriately. I believe they should be: *Cluster ID*, *Provider*, and *OpenShift version* but confirm with ENG/QE.

**Comment by aburdenthehand** [comment]
@bgaydosrh I've provided some feedback but I think the cart has been put before the horse. Peer Review should be the final step before being merged, after it has had QE ack. Is there a reason why this card skipped QE review?
Also this PR does not cover CNV-2294 at all, so you should probably remove it from the PR title to avoid confusion.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Thanks, changed to "overview dashboard" that describes cluster information (no CNV references).

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Simplified/clarified topic to: 

Access the overview dashboard, which captures high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-title} web console.

The overview dashboard provides various cluster information, captured in individual dashboard cards.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Noted, thanks!

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Good calls. Thanks all. Done.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Done.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks, made all changes. I am going with overview dashboard, using sentence casing as it seems this is how we refer to other UIs, as opposed to products which seem to be capped.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Made all changes. @rawagner can you confirm these labels are as @aburdenthehand states?

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Done. Thanks.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Moved the states up to the top of the card summary, didn't like them in a NOTE. 

Question for @rawagner --- Is the state called "progress" or "in progress"? Thanks.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Good suggestion. Done.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks Vikram, corrected typo. 

Ashley suggested not saying "card" numerous times, so I added an intro sentence and eliminated the articles and "card":

_The overview dashboard consists of the following cards:_

To me, it looks like if you add the article you also have to add "card". For example, "The Details card" not "The Details". 

Your call @aburdenthehand.

**Comment by bgaydosrh** [comment]
> It seems that this doc should go directly to openshift docs, not CNV docs. Overview Dashboard is part of core Openshift UI. When CNV is installed, it will extend Overview Dashboard for CNV specific parts like CNV subsystem health and VMs in Inventory Card

@rawagner Ashley and I will be reusing this overview dashboard doc, so the user should see the same info regardless of whether CNV is installed or not. Thanks!

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks I have amended this line to read:

** Virtual machines (only available if Container Native Virtualization installed)

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks, changed to:

** Bare metal hosts in the cluster, listed according to their state (only available in metal3 environment).

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Changed to:

* *Inventory* details number of resources and associated statuses. It is helpful when intervention is required to resolve problems, including information about:
.....

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Changed to:

* *Health* summarizes the current health of the cluster as a whole, including relevant alerts and descriptions. If more than one subsystem is present, click *See All* to view the status of each subsystem.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks! Yes, after reviewing the video you are correct. All changes synced with that view of the UI.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Hi Andrew,

I synced up file name and ID on the other file. cnv-dashboard looks fine to me (the file is cnv-dashboard.adoc). See below. Am I missing something? Thanks,

Bob

[id="cnv-dashboard"]
= Using the overview dashboard to get cluster information
include::modules/cnv-document-attributes.adoc[]
:context: cnv-dashboard
toc::[]

Access the overview dashboard, which captures high-level information about the cluster, by clicking *Home > Dashboards > Overview* from the {product-title} web console.

The overview dashboard provides various cluster information, captured in individual dashboard _cards_.

include::modules/cnv-web-console-overview-dashboard.adoc[leveloffset=+1]

**Comment by bgaydosrh** [comment]
This PR is ready for QE Review. I am tagging Ariel Chernetsky in Jira, I can't find a way to tag here.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (aburdenthehand) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: I meant the title of the assembly. In this case if the assembly is 'Using the overview dashboard to get cluster information' then I would give the id and filename something like 'cnv-using-overview-dashboard'. It's not a dealbreaker but it does make it much easier to find an assembly/module in the repo when you're looking at something in the built doc (subsequent releases/bugs).

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (aburdenthehand) [reply modules/cnv-web-console-overview-dashboard.adoc]: Much improved.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (rawagner) [reply modules/cnv-web-console-overview-dashboard.adoc]: yes, @aburdenthehand is correct

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (rawagner) [reply modules/cnv-web-console-overview-dashboard.adoc]: we just show an icon, there is no text for the status. But i guess `in progress` is better in docs

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Done, thanks guys.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Cool, thanks. Done.

File: modules/cnv-web-console-overview-dashboard.adoc
Comment (bgaydosrh) [reply modules/cnv-web-console-overview-dashboard.adoc]: Thanks. Done.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Given the time left in the release, I think I'll keep the assembly title as cnv-dashboard, but I will make a task to change the title after we release.

**Comment by bgaydosrh** [comment]
All content changes are now in. Needing QE review.

Preview build pages are here along with description of changes:

http://file.bos.redhat.com/bgaydos/092619/cnv/cnv_install/installing-container-native-virtualization.html - Removed the note at the referring to the cnv web console 

http://file.bos.redhat.com/bgaydos/092619/cnv/cnv_users_guide/cnv-edit-vms.html - 
Commented out Editing VM Web module based on absent functionality. Release note indicating this change coming today.

http://file.bos.redhat.com/bgaydos/092619/cnv/cnv_users_guide/cnv-accessing-vm-consoles.html - 
Note amended Step # 2 in "Connect to virtual machine with the web console" - it now reads "Click Workloads > Virtual Machines".

http://file.bos.redhat.com/bgaydos/092619/cnv/cnv_users_guide/cnv-dashboard.html - New assembly and module detailing existence of OpenShift Container Platform dashboard.

File: cnv/cnv_users_guide/cnv-dashboard.adoc
Comment (vikram-redhat) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Hey @bgaydosrh - doing this after the release means that the link to this assembly will change, if you change the ids and filenames. These will then require a redirect. It might be best to do this before.

Comment (bgaydosrh) [reply cnv/cnv_users_guide/cnv-dashboard.adoc]: Gotcha, I will do this tomorrow.

**Pull request merged**
